src/web/src/App.vue

```diff
@@ -56,6 +56,15 @@ export default {
       );
     },
   },
+  computed: {
+    colorSchemeMeta() {
+      return this.isFollowingDeviceColor()
+              ? "light dark"
+              : this.$cookies.get(COOKIE_DARK_MODE) === "true"
```

Comment:
Is this the current way we are checking for dark mode? I remember that we were trying to move away from putting things like this in cookies.

Reply:
I think this part does not involve dark mode checking...  
The `"light dark"` string is for `<meta>` tag.  It enables the UA to choose the correct UA style for light/dark mode, i.e. the light/dark scrollbars in this case.  

The dark mode decision is at L29-L58 which is not listed here.

Reply:
> I think this part does not involve dark mode checking...
> The `"light dark"` string is for `<meta>` tag. It enables the UA to choose the correct UA style for light/dark mode, i.e. the light/dark scrollbars in this case.
> 
> The dark mode decision is at L29-L58 which is not listed here.

I saw the cookie check so I wanted to be sure since I know we were moving aware from storing things that we need in order to run properly in the cookies